app.js

```diff
@@ -1,10 +1,14 @@
 const express = require("express");
 const app = express();
-const {getApi} = require("./controllers/app.controllers")
+const { getApi, getTopics } = require("./controllers/app.controllers");
+const { wrongPathHandler } = require("./error/app.error");
 
+app.use(express.json());
 
-app.get("/api", getApi)
+app.get("/api", getApi);
 
+app.get("/api/topics", getTopics);
 
+app.use(wrongPathHandler);
```

Comment:
I like how you've extracted all the controllers and error handlers out, very neat 🥇